### PR TITLE
Fixes #648

### LINKED ISSUE #648: remove 'new' from section description
I think it may be helpful to keep the released/added date on the content page but I think it would be more efficient and a better use of that description to remove the 'new in..' announcement from the section description that's displayed on the index page:

Screen Shot 2019-09-11 at 7 32 08 PM
Screen Shot 2019-09-11 at 7 32 15 PM

### PR DESCRIPTION
**Pull request by bricakeld**
Post-merge checklist:

The master branch is auto-deployed to `dash.plot.ly`.
Once you have merged your PR, wait 5-10 minutes and check dash.plot.ly 
to verify that your changes have been made.

- [x] I understand

### CODE REVIEW

File: tutorial/chapter_index.py [hunk 1]
```diff
@@ -167,7 +167,7 @@
         'url': '/datatable',
         'content': dash_table_index.layout,
         'name': 'Dash DataTable',
-        'description': '(New! Released Nov 2, 2018) The Dash DataTable is our latest and ' \
+        'description': '(Released Nov 2, 2018) The Dash DataTable is our latest and ' \
```
Comment (alexcjohnson) [inline tutorial/chapter_index.py]: For table let's actually take out the whole first sentence - it's a whole year old now so the release date isn't that helpful, it's no longer our latest component, and it's got a lot of competition for "most advanced" at this point.

File: tutorial/chapter_index.py [hunk 2]
```diff
@@ -178,7 +178,7 @@
         'url': '/cytoscape',
         'content': dash_cytoscape_index.layout,
         'name': 'Dash Cytoscape',
-        'description': '(New! Released Feb 5, 2019) Dash Cytoscape is our new network ' \
+        'description': '(Released Feb 5, 2019) Dash Cytoscape is our new network ' \
```
Comment (alexcjohnson) [inline tutorial/chapter_index.py]:
```suggestion
        'description': '(Released Feb 5, 2019) Dash Cytoscape is our network ' \
```

**Comment by alexcjohnson** [comment]
Thanks for the PR @bricakeld!

Let's also take out "New in dash v0.22.0!"
https://github.com/plotly/dash-docs/blob/974f409e32c80a0b8c7eab0d3bd76ae11d5f1a92/tutorial/chapter_index.py#L252

and "New in Dash 1.0!"
https://github.com/plotly/dash-docs/blob/974f409e32c80a0b8c7eab0d3bd76ae11d5f1a92/tutorial/chapter_index.py#L1046

File: tutorial/chapter_index.py (hunk 2 above)
Comment (bricakeld) [reply tutorial/chapter_index.py]: Thanks for the comments, I'll make the changes and update this PR

**Pull request merged**